test/system/800-system-reset.bats

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# tests for podman system reset
+#
+
+load helpers
+
+function setup() {
+    if is_remote; then
+        skip "only applicable when running non remote"
+    fi
+
+    basic_setup
+}
+
+
+function teardown() {
+    basic_teardown
+}
+
+
+@test "podman system reset - storage options" {
+        rr=$PODMAN_TMPDIR/containers
+        mkdir -p $rr
+
+        gr=$PODMAN_TMPDIR/.local/share/containers/storage
+        mkdir -p $gr
+
+        run_podman system reset -f --run-root=${rr} --graph-root=${gr}
+        dir=/home/$(id -un)/.config/containers/storage.conf
```

Comment:
I don't believe Ed is going to like this. You might be able to use a STORAGE_CONF environment variable for this
Or at least set $HOME=PODMAN_TMPDIR/home or something like that.

Final Nit is storage.conf is a file not a dir.  Would more sense to say storage_file=/home...

Reply:
Thanks for the ping. Dan is correct, I would really not appreciate having my podman environment clobbered by running a test. Also, `/home/$(id -un)` doesn't work for root. Also, I too am confused by your use of `$dir` and the `[ -d` below. I need to take a pretty close look at this, and will try to do so today.

Reply:
I was unsure of how to approach this, is it possible this same test could work as integration? I put it in system due to the nature of modifying storage.conf. This creates a similar issue as scp does with modifying user data.